api/src/main/java/org/apache/cloudstack/backup/Backup.java

```diff
@@ -125,9 +139,17 @@ public Long getSize() {
             return size;
         }
 
+        public void setDeviceId(Long deviceId) {
+            this.deviceId = deviceId;
+        }
+
+        public Long getDeviceId() {
+            return deviceId;
+        }
+
         @Override
         public String toString() {
-            return StringUtils.join(":", uuid, path, type, size);
+            return StringUtils.join(":", uuid, path, type, size, deviceId);
```

Comment:
This `toString` is used in many places? If not, we could rewrite it to use the `ReflectionToStringBuilderUtils`.